README.md

```diff
@@ -121,8 +121,8 @@ We used `create-react-app` to initialize this project, so it is configurated to
 Now that you are set up, open a terminal and:
 
 ```sh
-git clone https://github.com/Azordev/art-marketplace
-cd art-marketplace
+git clone https://github.com/Azordev/sencico
+cd sencico
```

Comment:
Seems like your branch need a rebase, you have old code

Reply:
@e-azocar can you confirm please?

Reply:
Yes, please rebase

Reply:
Yes, the branch is old, maybe is using the branch "master" or another branch